src/Faker/Provider/Color.php

```diff
@@ -113,4 +113,20 @@ public static function colorName()
     {
         return static::randomElement(static::$allColorNames);
     }
+
+    /**
+     * @example 'olive'
+     */
+    public static function htmlColorName()
```

Comment:
Not sure about this method, it effectively introduce an alias vor `safeColorName()` - is that intended?

Reply:
yeah, the idea was to have two methods that clearly express relation with web colors